src/lib/ngsi/Request.cpp

```diff
@@ -75,6 +75,7 @@ const char* requestType(RequestType rt)
   case Ngsi10SubscriptionsConvOp:                   return "Ngsi10SubscriptionsConvOp";
 
   case LogRequest:                                  return "Log";
+  case LogLevelRequest:                             return "LogLevel";
   case VersionRequest:                              return "Version";
```

Comment:
And, yes, I wrote "/v1/log/traces" a little quick, without checking the exact URLs:

    #define LOGT_COMPS_V0      2, { "log", "trace"                           }
    #define LOGTL_COMPS_V0     3, { "log", "trace",      "*"                 }
    #define LOG2T_COMPS_V0     2, { "log", "traceLevel"                      }
    #define LOG2TL_COMPS_V0    3, { "log", "traceLevel", "*"                 }
    #define LOGT_COMPS_V1      4, { "v1", "admin", "log", "trace"            }
    #define LOGTL_COMPS_V1     5, { "v1", "admin", "log", "trace",      "*"  }
    #define LOG2T_COMPS_V1     4, { "v1", "admin", "log", "traceLevel"       }
    #define LOG2TL_COMPS_V1    5, { "v1", "admin", "log", "traceLevel", "*"  }